docs/reference/overview/excel-add-ins-reference-overview.md

```diff
@@ -8,56 +8,38 @@ localization_priority: Priority
-  - [DataValidation](/javascript/api/excel/excel.datavalidation): An object that restricts user input to a range based on a variety of criteria.
-  - [RangeSort](/javascript/api/excel/excel.rangesort): Represents a object that manages sorting operations on a range.
+Review the following articles for information about important programming concepts:
+ 
+- [Fundamental programming concepts with the Excel JavaScript API](../../excel/excel-add-ins-core-concepts.md)
 
-- [Table](/javascript/api/excel/excel.table): Represents a collection of organized cells designed to make management of the data easy.
-  - [TableCollection](/javascript/api/excel/excel.tablecollection): A collection of tables in a workbook or worksheet.
-  - [TableColumnCollection](/javascript/api/excel/excel.tablecolumncollection): A collection of all the columns in a table.
-  - [TableRowCollection](/javascript/api/excel/excel.tablerowcollection): A collection of all the rows in a table.
-  - [TableSort](/javascript/api/excel/excel.tablesort): Represents an object that manages sorting operations on a table.
+- [Advanced programming concepts with the Excel JavaScript API](../../excel/excel-add-ins-advanced-concepts.md)
 
-- [Chart](/javascript/api/excel/excel.chart): Represents a chart object in a worksheet, which is a visual representation of underlying data.
-  - [ChartCollection](/javascript/api/excel/excel.chartcollection): A collection of charts in a worksheet.
+## Learn about API capabilities
 
-- [PivotTable](/javascript/api/excel/excel.pivottable): Represents an Excel PivotTable, which is a hierarchical grouping and presentation of data.
-  - [PivotTableCollection](/javascript/api/excel/excel.pivottablecollection): A collection of PivotTables in a worksheet.
+Review other articles in this section of the documentation for guidance about using the Excel JavaScript API to implement various types of functionality in your add-in. For example, learn about working with [events](../../excel/excel-add-ins-events.md), [charts](../../excel/excel-add-ins-charts.md), [ranges](../../excel/excel-add-ins-ranges.md), [tables](../../excel/excel-add-ins-tables.md), [worksheets](../../excel/excel-add-ins-worksheets.md), and more. Also here in this section of the documentation, you'll find guidance about Excel JavaScript API concepts such as [coauthoring in Excel add-ins](../../excel/co-authoring-in-excel-add-ins.md), [data validation](../../excel/excel-add-ins-data-validation.md), [error handling](../../excel/excel-add-ins-error-handling.md), and [performance optimization](../../excel/performance.md). See the table of contents for the complete list of available articles.
 
-- [Filter](/javascript/api/excel/excel.filter): Represents an object that manages the filtering of a table's column.
+For hands-on experience using the Excel JavaScript API to access objects in Excel, complete the [Excel add-in tutorial](../../tutorials/excel-tutorial.md). 
```

Comment:
Can we merge the SL section with this? Hands-on experience makes me think of Script Lab.